packages/features/eventtypes/lib/getPublicEvent.ts

```diff
@@ -137,7 +136,7 @@ export const getPublicEvent = async (
         username: users[0].username,
         name: users[0].name,
         weekStart: users[0].weekStart,
-        image: `${WEBAPP_URL}/${users[0].username}/avatar.png`,
+        image: `/${users[0].username}/avatar.png`,
```

Comment:
For public booking pages, I am using relative URL. There seems to be no need of using full URL and there isn't a quick way to get full URL without using orgBranding which requires the user to be logged in.

Using relative URL seems to be the ideal way to me.